armi/reactor/converters/axialExpansionChanger.py

```diff
@@ -190,9 +202,19 @@ def setAssembly(self, a, setFuel=True):
         setFuel : boolean, optional
             Boolean to determine whether or not fuel blocks should have their target components set
             This is useful when target components within a fuel block need to be determined on-the-fly.
+        Tinput2Thot: bool
```

Comment:
This variable name feels non-ideal, in comparison to ARMI's typical conventions. It also doesn't feel very intuitive to me, but just IMO.

Maybe suggest a change to something like `useTinputAsColdTemp` or something?

Reply:
I agree it goes against typical ARMI naming conventions and should be renamed. How about `expandFromTinputToThot`?

Reply:
I'm a little surprised the linter didn't catch the bad variable name! 
@opotowsky @john-science

Reply:
Seems like a reasonable name to me!

Reply:
https://github.com/terrapower/armi/pull/1342/commits/ad5d252d13c16a99b23ffc91f617a394ffa624e0